src/ui/components/hero/Hero.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react'
+import { Dimensions } from 'react-native'
+import FastImage from 'react-native-fast-image'
+import styled from 'styled-components/native'
+
+import { CategoryNameEnum } from 'api/gen'
+
+import { ImagePlaceholder } from '../../components/ImagePlaceholder'
```

Comment:
```suggestion
import { ImagePlaceholder } from 'ui/components/ImagePlaceholder'
```

Pareil pour les autres imports